canonical-web-and-design/maas-ui
# Add the cloud init option and input

## Done
- Added the checkbox and textarea for the cloud init script.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps
- Select a deployable machine in the machine list and use the take action menu to open the deploy form.
- Click on the new cloud init checkbox.
- The cloud init script text area should now be visible.

## Fixes

Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/2032.
